src/Series.cpp

```diff
@@ -770,20 +769,28 @@ cleanFilename(std::string const& filename, Format f)
 }
 
 std::function< bool(std::string const&) >
-matcher(std::string const& name, Format f)
+matcher(std::string const& seriesName, Format f)
 {
     switch( f )
     {
         case Format::HDF5:
         {
-            std::regex pattern(auxiliary::replace_last(name + ".h5$", "%T", "[[:digit:]]+"));
-            return [pattern](std::string const& filename) -> bool { return std::regex_search(filename, pattern); };
+            return [seriesName](std::string const& filename) -> bool
+            {
+                bool const matchEnd( filename.rfind(".h5") == filename.size() - 3u );
```

Comment:
We have a whole bunch of [string manipulation](https://github.com/openPMD/openPMD-api/blob/dev/include/openPMD/auxiliary/StringManip.hpp) coded up and tested already, if you want to save some time.